test/_meta.lua

```diff
@@ -0,0 +1,10 @@
+--- @meta
```

Comment:
No objection in this case, but reminder: avoid building types and features around any test utils that can possibly be moved into stdlib. We want to avoid fossilizing test utils that should actually be in the stdlib (or private "shared" functions when stdlib is questionable).

Reply:
This is for a busted function, which we borrow from LuaLS. Eventually these will be removed so we may need to provide them all anyway.

> ... be moved into stdlib. 

I assume/hope you mean this with the condition that they don't follow any API contract?

Reply:
> Eventually these will be removed so we may need to provide them all anyway.

Is there a tracking issue for that? Would we expose them in the stdlib? (just curious)

> I assume/hope you mean this with the condition that they don't follow any API contract?

If we don't want to follow an API contract we can move them into "shared" or some other internal-only place (and we can always graduate to stdlib incrementally as needed). 

The main idea is just to avoid duplicat in our test util functions, which are largely redundant.

Reply:
> Is there a tracking issue for that? Would we expose them in the stdlib? (just curious)

Not really. I think there's an issue for the luv definitions which have the same fate.

> The main idea is just to avoid duplication in our test util functions, which are often redundant and/or potentially useful for non-test code internally or even in the public stdlib.

AFAIK, the test utils have much less duplication now. The duplication was a result of us running busted in PUC Lua/LuaJIT, but now we use `nvim -l` which allowed us to consolidate a bunch of stuff.